packages/node_modules/@webex/plugin-meetings/src/meeting/index.js

```diff
@@ -4406,4 +4433,92 @@ export default class Meeting extends StatelessWebexPlugin {
       }
     );
   }
+
+
+  /**
+   *
+   * Handle logging various things for meetings. Such as audio, video and device details.
+   * @returns {Promise} resolves to an array of DeviceInfo
+   * @memberof Meetings
+   */
+  handleMediaLogging = async () => {
+    LoggerProxy.logger.log('Meeting:index#handleMediaLogging');
+    await this.handleDeviceLogging();
+  }
+
+  /**
+   * Handle audio player logging, dumps details such as autoGainControl, echoCancellation
+   * volume and deviceId
+   * @returns {Promise} resolves empty promise
+   * @memberof Meetings
+   */
+  handleAudioLogging = async () => {
+    if (this.audioTrack) {
+      const {label} = this.audioTrack;
+      const {
+        autoGainControl, echoCancellation, noiseSuppression, deviceId
+      } = this.audioTrack.getSettings();
+
+      LoggerProxy.logger.log('Meeting:index#handleAudioLogging');
+      LoggerProxy.logger.group('audio');
+      LoggerProxy.logger.log('deviceId=', deviceId);
+      LoggerProxy.logger.log('label=', label);
+      LoggerProxy.logger.log('noiseSuppression=', noiseSuppression);
+      LoggerProxy.logger.log('echoCancellation=', echoCancellation);
+      LoggerProxy.logger.log('autoGainControl=', autoGainControl);
+      LoggerProxy.logger.groupEnd('audio');
+    }
+
+    return this;
+  }
+
+  /**
+   * Handle video player logging, dumps video details such as
+   * frameRate, resolution and deviceId
+   * @returns {Promise} resolves empty promise
+   * @memberof Meetings
+   */
+  handleVideoLogging = () => {
+    if (this.videoTrack) {
+      const {
+        deviceId, frameRate, aspectRatio, height, width
+      } = this.videoTrack.getSettings();
+
+      LoggerProxy.logger.log('Meeting:index#handleVideoLogging');
+      LoggerProxy.logger.group('video');
+      LoggerProxy.logger.log('deviceId=', deviceId);
+      LoggerProxy.logger.log('aspectRatio=', aspectRatio);
+      LoggerProxy.logger.log('frameRate=', frameRate);
+      LoggerProxy.logger.log('width=', width);
+      LoggerProxy.logger.log('height=', height);
+      LoggerProxy.logger.groupEnd('video');
+    }
+
+
+    return this;
```

Comment:
Why are you returning `this`?

Reply:
eslint was asking for a return value and usage of `this` in class member.